public/app/features/alerting/unified/utils/amroutes.ts

```diff
@@ -198,10 +198,10 @@ export const stringToSelectableValue = (str: string): SelectableValue<string> =>
 export const stringsToSelectableValues = (arr: string[] | undefined): Array<SelectableValue<string>> =>
   (arr ?? []).map(stringToSelectableValue);
 
-export const mapSelectValueToString = (selectableValue: SelectableValue<string>): string | undefined => {
+export const mapSelectValueToString = (selectableValue: SelectableValue<string>): string | undefined | null => {
```

Comment:
with this change , is it possible to return `undefined`?

Reply:
Good catch! It won't, only `null` or `string` would be returned